bmq-sdk/src/test/java/com/bloomberg/bmq/resultcodes/AckResultChecks.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright 2024 Bloomberg Finance L.P.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.bloomberg.bmq.resultcodes;
+
+import com.bloomberg.bmq.ResultCodes;
+
+public class AckResultChecks {
```

Comment:
Do such classes need to be in separate files?

Reply:
I've looked one more on the structure, it is not needed, at the time of applying changes it has been easier to keep them in such a structure, but, I've consolidated checks with particular test classes, the only one - generic, I kept separate since it is related to all of them.

Let me know what do you think about that!

Reply:
Missed this comment.